### DISCUSSION TITLE
Feed update issue on shared host

### DISCUSSION THREAD

**Discussion by bigl00z3** [FreshRSS/FreshRSS#4640] [Q&A] [answered]
I'm having issues with the automatic update of my feed. I have read the [doc ](https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron) and tried to search in the discussion list for a solution without any success so far.

**As background info:**

- My setup is on a shared hosting at [Infomaniak.com](https://www.infomaniak.com/en) where I don't have access to Cron but to their "Task Planner" ( [screenshot](https://bigloose.com/nb/temp/TaskPlanner.jpg) )
- I am mostly accessing my feed on my phone or laptop via an external app (API ticked and correctly working) which means that I can't just click in the web interface to manually update every time.

**Pre tests in my browser:**

- Calling the initial url   `https://mywebsite/installfolder/i/?c=feed&a=actualize ` based on the doc is giving me an error 
- But `https://mywebsite/installfolder/p/i/?c=feed&a=actualize` (added the "/p/" before the i) updates the feed correctly. However this only works if I'm already logged in to my account.

In the Infomaniak Task Planner I have tried to set the task with the second url that works in my browser, and I get an http response "200" meaning the task has been accomplished successfully. 
But the feeds are not updated. 
I imagine that it is because I am not logged in my FreshRSS account. I have tried to tick the box "This URL is password-protected" and enter my account details but without any success.

My next step in trying to progress and based on some of the comments I read [here](https://lokoyote.eu/freshrss-probleme-de-cron/) has been to set a task with the Token url (activated in my profile) which gives me an url looking like `https://mywebsite/installfolder/p/i/?c=feed&a=actualize&token=542345872345734&hours=168`
Here I have tried with the "&hours=168" and also without it. In both cases the feeds are not updating.

Any other ideas on what to test? 
Anything jumps to your eyes as wrong in what I have tested?

Any help would be highly appreciated

Nico

**Comment by Alkarex** [comment, marked as answer]
Hello,
See https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron

Try
`https://mywebsite/installfolder/p/i/?c=feed&a=actualize&ajax=1&maxFeeds=30&user=alice&token=123`

**Reply by bigl00z3** [reply]
Hello @Alkarex 

I had checked on that page but had not tried (my bad) this combination. All good now it's working perfectly.

Thanks a lot

Nico
